### DISCUSSION TITLE
Keyball61のトラックボール（右側）のパットが剥離してしまいました。

### DISCUSSION THREAD

**Discussion by hisui-studio** [Yowkees/keyball#258] [Q&A] [answered]
画像の位置（左から二番目のシャンパ）のパットにブリッジをしようとしたところパットが剥離してしまいました。
他のQ&Aを見たところ他のところを繋げれば修復可能なようですが、どこを繋げれば良いのか教えていただきたいです。

解決方法をご教示頂けますと幸いです。

![image](https://github.com/Yowkees/keyball/assets/113280738/2ce11068-0381-4db7-97e4-a6cdf55c5b4d)

**Comment by Yowkees** [comment, marked as answer]
下記画像の緑点２点をエナメル線かゴム被覆導線で接続していただければ、赤丸部のジャンパーが成功したのと同じ回路となり、修正できます。
お手数ですがご対応お願いします。
![KB67R_jampfix](https://github.com/Yowkees/keyball/assets/84209772/9a9e0bec-c6d4-45e0-8776-a5ba8c39a937)

**Reply by hisui-studio** [reply]
対応して頂きありがとうございます。

画像のように繋げたところOLED上部が異常に発熱するようになってしまいました。

実は左側もこの症状に悩まされており、ハンダのつけ直しなど原因を探っていましたが分からずにいます。

解決方法をご提示頂けますと幸いです。

**Reply by hisui-studio** [reply]
ProMicroの取り付け向きのミスが原因だったようです。
対応して頂きありがとうございました！

**Reply by Yowkees** [reply]
返信が遅くなり申し訳ありませんでした。
ご自身で解決いただきありがとうございました！
今後ともどうぞよろしくお願いいたします。
